codeforamerica/vita-min
# FYST 185 Add pages for public school contributions

## Link to pivotal/JIRA issue
https://codeforamerica.atlassian.net/browse/FYST-185

## Is PM acceptance required?
- Yes - don't merge until JIRA issue is accepted!

## What was done?
Pages have been added to arizona flow for public school contributions. It follows the format of unemployment controller where there is an index of all contribution where the client can edit and add more contributions
 
## How to test?
Go on the heroku link and fill out the flow for arizona. Make sure to change the session toggles for heroku to be during intake time. 

## Screenshots (for visual changes)
<img width="749" alt="Screenshot 2024-08-05 at 12 27 44 PM" src="https://github.com/user-attachments/assets/ed3b039e-ae58-4d86-88ad-01a0a0a2e2f7">
<img width="248" alt="Screenshot 2024-08-05 at 12 32 11 PM" src="https://github.com/user-attachments/assets/ca2e0f2f-75f0-413e-8e92-4ad82c55aa86">
<img width="718" alt="Screenshot 2024-08-05 at 12 35 40 PM" src="https://github.com/user-attachments/assets/98a36da4-3191-4dc6-8993-56d707cf2b2d">
